src/services/RequestService.js

```diff
@@ -1,32 +1,48 @@
 import axios from "axios";
+import { config as configUrl } from "../config";
 
-const config = () => ({
+const config = {
   headers: {
     "Access-Control-Allow-Origin": "*",
     "Content-type": "application/json; charset=UTF-8",
   },
-});
+};
+
+let authConfig = {
+  headers: {
+    "Access-Control-Allow-Origin": "*",
+    "Content-type": "application/json; charset=UTF-8",
+    Authorization: `TOKEN ${sessionStorage.getItem("Token")}`,
+  },
+};
 
 const RequestService = {
   get: async (url, callback, failureCallback) => {
     axios
-      .get(url, config())
+      .get(url, authConfig)
       .then((response) => {
         callback(response);
       })
       .catch((err) => failureCallback && failureCallback(err));
   },
   post: async (url, body, callback, failureCallback) => {
+    let postConfig = authConfig;
+
+    // if post request is accessing login (auth) endpoint
+    // use generic config headers without Authorization info (since there isn't any yet)
+    const isLoginPost = url === configUrl.backendUrlAuth;
+    if (isLoginPost) postConfig = config;
```

Comment:
Could you please wrap this block with {}

Reply:
This is a one line if statement that just doesn't need the brackets. Do you want to add it for preference/readability?

Reply:
This is usually a best practice for readability.

Reply:
Updated